packages/template-retail-react-app/app/components/order-summary/index.jsx

```diff
@@ -61,6 +61,7 @@ const CartItems = ({basket}) => {
             <Box>
                 <Button
                     variant="link"
+                    color="blue.700"
```

Comment:
Since Order Summary section does not have whilte background, the background is a gray color. We need a darker blue to satisfy the contrast requirement
![image](https://github.com/SalesforceCommerceCloud/pwa-kit/assets/52219283/901727b3-2ad3-4f25-935d-2a32e75db492)